src/app.jsx

```diff
@@ -1,11 +1,12 @@
 import React from "react";
 import ReactDOM from "react-dom";
-import Calculator from "./calculator.jsx";
+import { Calculator, FunctionCalculator } from "./calculator";
```

Comment:
right now ive been doing destructuring with imports, but there is a lot of strong evidence out there that suggests you shouldn't and instead import 1 at a time, refecering the exact file.

Read: https://lacke.mn/reduce-your-bundle-js-file-size/